### ISSUE TITLE
Can't remove permission via command 

### ISSUE THREAD

**Issue by sunnamed434** [RocketModFix/RocketModFix#107]
![image.png](https://github.com/user-attachments/assets/93a415ea-e345-441e-a8c1-fff3c766a108)

Reported by `jonnygamingtv` in discord

**Comment by JonnygamingTv** [comment]
needed lowercase h, however it should say "group was not found" instead of "player was not found"

**Comment by sunnamed434** [comment]
Yes, it's a bug, and it should show that `player was not found` instead.

> 
> Also unsure why it makes the groupids lowercase but not the group input
> Like:
> Helper = can add by /p add helper, but not Helper
> helper = can add by /p add HElper

Nice catch, need to check in code what's going on there, probably just need to make a variable for that case, can't say much right now, but might be a bug also.
